.scripts/curl.sh

```diff
@@ -1,12 +1,27 @@
 # Test POST method for createAccount
-curl -d'{"username":"Bob","firstName":"Bob","lastName":"Demo","email":"bobdemo@gmail.com", "verified":true, "createTime":"2017-07-21T17:32:28Z","updateTime":"2017-07-21T17:32:28Z","description":"Test account for Bob", "password":"PASSWORD"}' -H "Content-Type: application/json" -X POST http://localhost:8080/v1/accounts/ -i
+curl -d'{"username":"Alice","firstName":"Alice","lastName":"Demo","email":"alice@gmail.com", "emailStatus":1, "description":"Test account for Alice", "password":"PASSWORD"}' -H "Content-Type: application/json" -X POST http://localhost:8080/v1/accounts/ -i
+echo ""
```

Comment:
why do we want to print an empty line here and below?

Reply:
`curl -i` will append the message to the terminal without starting a new line.  Doing that is just for the ease of checking the result.

Reply:
You can add the below code at the beginning of the file. It can log every command it is going to execute.
```
#!/bin/sh
set -x
```